schemas/webpack.d.ts

```diff
@@ -0,0 +1,364 @@
+        and?: RuleSetRule[];
+        exclude?: RuleSetRule;
+        include?: RuleSetRule;
+        not?: RuleSetRule[];
+        or?: RuleSetRule[];
+        test?: RuleSetRule;
+    }
+
+    interface RuleSetUseItemObject {
+        loader?: string;
+        options?: RuleSetQuery;
+        query?: RuleSetQuery;
+    }
+
+    interface Output {
+        // Add a comment in the UMD wrapper.
+        // String: Append the same comment above each import style.
+        // OutputAuxiliaryComment: Set explicit comments for `commonjs`, `commonjs2`, `amd`, and `root`.
+        auxiliaryComment?: string | OutputAuxiliaryComment;
+
+        // The filename of non-entry chunks as relative path inside the `output.path` directory.
+        chunkFilename?: string;
+
+        // This option enables cross-origin loading of chunks.
+        crossOriginLoading?: boolean | "anonymous" | "use-credentials";
+
+        // Similar to `output.devtoolModuleFilenameTemplate`, but used in the case of duplicate module identifiers.
+        devtoolFallbackModuleFilenameTemplate?: string | Function;
+
+        // Enable line to line mapped mode for all/specified modules. Line to line mapped mode uses a simple SourceMap where each line of the generated source is mapped to the same line of the original source. It’s a performance optimization. Only use it if your performance need to be better and you are sure that input lines match which generated lines.
+        // Boolean: `true` enables it for all modules (not recommended)
+        // Object: An object similar to `module.loaders` enables it for specific files.
+        devtoolLineToLine?: boolean | OutputDevtoolLineToLine;
+
+        // Filename template string of function for the sources array in a generated SourceMap.
+        devtoolModuleFilenameTemplate?: string | Function;
+
+        // Specifies the name of each output file on disk. You must **not** specify an absolute path here! The `output.path` option determines the location on disk the files are written to, filename is used solely for naming the individual files.
+        filename?: string;
+        hashDigest?: string;
+        hashDigestLength?: number;
+        hashFunction?: string;
+
+        // The filename of the Hot Update Chunks. They are inside the output.path directory.
+        hotUpdateChunkFilename?: string;
+
+        // The JSONP function used by webpack for async loading of hot update chunks.
+        hotUpdateFunction?: string;
+
+        // The filename of the Hot Update Main File. It is inside the `output.path` directory.
+        hotUpdateMainFilename?: string;
+
+        // The JSONP function used by webpack for async loading of chunks.
+        jsonpFunction?: string;
+
+        // If set, export the bundle as library. `output.library` is the name.
+        library?: string | string[];
+
+        libraryTarget?: OutputLibraryTarget;
+
+        // The output directory as **absolute path** (required).
+        path?: string;
```

Comment:
That's correct, but in the tests that you can find in this repository, there is a lot of [example](https://github.com/webpack/webpack/blob/54aa3cd0d6167943713491fd5e1110b777336be6/test) without path in output object, so I am not so sure about it. But on [website](http://webpack.github.io/docs/configuration.html#output-path) it's required.